### PR TITLE
replace setTimeout, setInterval to QUnitGlobals.wait, use QUnitGlobals.waitForIframe for waiting iframe load (close #280)

### LINKED ISSUE #280: Replace all setTimeout in the client tests to the QUnitGlobals.wait

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @LavrovArtem @VasilyStrelyaev

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
Saucelabs is temporary unavailabe. We can track the state [here](http://status.saucelabs.com/)

File: test/client/fixtures/sandbox/event/focus-blur-change-test.js [hunk 1]
```diff
@@ -664,12 +667,11 @@ if (window.HTMLInputElement.prototype.setSelectionRange) {
...
-            startNext();
-        }, 200);
+        window.QUnitGlobals.wait(checkFocus)
+            .then(function () {
+                strictEqual(document.activeElement === input2, needFocus);
+                startNext();
+            });
     });
 
     asyncTest('setSelectionRange() called by some event handler when browser window is on background', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/event/focus-blur-change-test.js]:
```
     asyncTest('setSelectionRange() called by some event handler when browser window is in background', function () {
```

File: test/client/fixtures/sandbox/event/message-test.js [hunk 2]
```diff
@@ -51,37 +46,36 @@ asyncTest('onmessage event', function () {
...
+    window.QUnitGlobals.waitForIframe(iframe)
+        .then(function () {
+            eval(processScript('window.onmessage = onMessageHandler;', true, false));
+            window.addEventListener('message', onMessageHandler);
+            eval(processScript('iframe.contentWindow.postMessage(\'\', \'*\')', true, false));
+        });
+    document.body.appendChild(iframe);
 });
 
 asyncTest('crossdomain post messages between diffferen windows', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/event/message-test.js]:
```
 asyncTest('cross-domain post messages between different windows', function () {
```

File: test/client/fixtures/sandbox/iframe-test.js [hunk 3]
```diff
@@ -68,87 +68,94 @@ asyncTest('element.setAttribute', function () {
...
+            /* eslint-enable no-unused-vars */
+            ok(!iframeLoadingEventRaised);
+            iframeSandbox.off(iframeSandbox.IFRAME_READY_TO_INIT_EVENT, handler);
+            container.parentNode.removeChild(container);
+            start();
+        });
+    document.body.appendChild(container);
 });
 
 asyncTest('the AMD module loader damages proxing an iframe without src (GH-127)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/iframe-test.js]:
```
 asyncTest('the AMD module loader damages proxying an iframe without src (GH-127)', function () {
```

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by miherlosev** [comment]
FPR

**Comment by miherlosev** [comment]
ping @churkin @LavrovArtem

**Comment by LavrovArtem** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**